ouroboros-consensus/changelog.d/scriv.ini

```diff
@@ -0,0 +1,6 @@
+[scriv]
+format = md
+insert_marker = Changelog entries
+md_header_level = 2
+version = literal: ouroboros-consensus.cabal: version
+categories = Patch, Non-Breaking, Breaking
```

Comment:
I am unsure what the usual best practice is, but how about specifying all configuration options here even if we use the default values? Hopefully that would prevent breakage if the default values (as built in to `scriv`) change at some point, and IMO it makes it easier to see at a glance what the total configuration is without having to consult the documentation

Reply:
Done